Products/ZenModel/DeviceClass.py

```diff
@@ -357,12 +356,12 @@ def moveDevices(self, moveTarget, deviceNames=None, REQUEST=None):
         target = self.getDmdRoot(self.dmdRootName).getOrganizer(moveTarget)
         if isinstance(deviceNames, basestring): deviceNames = (deviceNames,)
         targetClass = target.getPythonDeviceClass()
-        numExports = 0
+        moved_devices_amount = 0
         for devname in deviceNames:
             devicewasExported = self._moveDevice(devname, target, targetClass)
             if devicewasExported:
-                numExports += 1
-        return numExports
+                moved_devices_amount += 1
+        return moved_devices_amount
```

Comment:
Just a nitpick, but please use "count" instead of "amount".  Both words have basically the same meaning, but "count" is the more common (and recognizable) word to use.

Reply:
Changed the `moved_devices amount` to the `moved_devices_count` according to your suggestion.